browse-scaler/serverless.yml

```diff
@@ -0,0 +1,53 @@
+service: browse-scaler-${self:provider.stage}
```

Comment:
Don't' we need this as a PR on EDSC since this inf is meant to go on that account?